### PR TITLE
feat: Credential Management - jwt vc support

### PR DESCRIPTION
**Pull request by mishasizov-SK**
Linked issue: https://github.com/trustbloc/vcs/issues/757

Signed-off-by: Mykhailo Sizov <mykhailo.sizov@securekey.com>

### CODE REVIEW

**Review by fqutishat — APPROVED** [review]

File: component/profile/reader/file/creator.go [hunk 1]
```diff
@@ -22,12 +22,18 @@ import (
...
 	vcsverifiable.Ed25519Signature2018:        crypto.Ed25519VerificationKey2018,
 	vcsverifiable.JSONWebSignature2020:        crypto.JSONWebKey2020,
 	vcsverifiable.EcdsaSecp256k1Signature2019: crypto.EcdsaSecp256k1VerificationKey2019,
 	vcsverifiable.BbsBlsSignature2020:         crypto.Bls12381G1Key2020,
+	// JWT
+	vcsverifiable.EdDSA:  crypto.JSONWebKey2020,
+	vcsverifiable.ES256K: crypto.JSONWebKey2020,
+	vcsverifiable.ES256:  crypto.JSONWebKey2020,
+	vcsverifiable.ES384:  crypto.JSONWebKey2020,
+	vcsverifiable.PS256:  crypto.JSONWebKey2020,
```
Comment (mishasizov-SK) [inline component/profile/reader/file/creator.go:36]: Please pay extra attention on this part of code - it's mapping between signature type and did verification method type. It's used [here](https://github.com/trustbloc/vcs/pull/827/files#diff-6e7fd18916da9184e146724584d2202fbe95e42bf84f7900252d82dcd94131d8L208). 
I'm not sure whether I'm right.

File: test/bdd/fixtures/profile/profiles.json [hunk 2]
```diff
@@ -172,6 +229,25 @@
...
+        "id": "v_myprofile_jwt",
+        "name": "v_myprofile_jwt",
+        "organizationID": "test_org",
+        "url": "https://test-verifier.com",
+        "active": true,
+        "checks": {
+          "credential": {
+            "format": ["jwt"],
+            "proof": true,
+            "status": false
```
Comment (rolsonquadras) [inline test/bdd/fixtures/profile/profiles.json:244]: We should add some tests with `status = true`.

Comment (mishasizov-SK) [reply test/bdd/fixtures/profile/profiles.json:244]: sure, will do

Comment (mishasizov-SK) [reply test/bdd/fixtures/profile/profiles.json:244]: @rolsonquadras The VC status check is yet to be implemented, but it's not ready for now. 
I added it as a subpoint in this ticket https://github.com/trustbloc/vcs/issues/757

**Review by vkubiv — APPROVED** [review]

**Pull request merged**